version.xml

```diff
@@ -1,7 +1,7 @@
 <!-- This file may be overwritten by automation. -->
 <Project>
   <PropertyGroup>
-    <KoreBuildChannel>dev</KoreBuildChannel>
+    <KoreBuildChannel>rel/2.0.0</KoreBuildChannel>
```

Comment:
Universe will override the version of KoreBuild and the 'dev' channel of KoreBuild will be moving to .NET Core 2.1 soon